src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversion.cs

```diff
@@ -357,8 +357,13 @@ internal MethodSymbol Method
                         return analysis.Operator;
                     }
 
-                    if (uncommonData is DeconstructionUncommonData deconstruction
-                        && deconstruction.DeconstructMethodInfo.Invocation is BoundCall call)
+                    if (uncommonData is DeconstructionUncommonData {
+                        DeconstructMethodInfo:
+                        {
+                            Invocation: BoundCall { } call
+                        }
+                    } deconstruction
+)
```

Comment:
📝 This parenthesis should have been placed at the end of the preceding line.

Reply:
shouldn't we depend on the formatter to handle these? A fresh recursive pattern node should be at least formatted properly, if it doesn't include additional trivia.

Reply:
Verified fixed